ui/panels-plugin/src/model/thresholds.ts

```diff
@@ -16,10 +16,10 @@ import { zip } from 'lodash-es';
 
 // TODO (sjcobb): pull threshold colors from perses charts theme
 export const ThresholdColors = {
```

Comment:
I'll update these to pull from the MUI theme in a separate PR after Eunice's design system changes are merged

Reply:
Dang I forgot to do this, I'll follow up in a separate PR since I'll need to test the changes with TimeSeriesChart threshold lines as well.